style.css

```diff
@@ -0,0 +1,28 @@
+#title{
+    color: rgb(66, 57, 57);
+}
+#description{
+    font-size: 16px;
+}
+#name{
+    width: 200px;
+    height: 10px;
+    font-size: 16px;
+    color: rgb(194, 72, 72);
+}
+#email{
+    width: 200px;
+    height: 10px;
+    font-size: 16px;
+    color: rgb(194, 72, 72);
```

Comment:
Choose Good colors for most of the text don't use rainbow-type colors it can affect your user's interference.

Reply:
[OPTIONAL] Well done 🎉 but how about you formulate these more as suggestions than orders, try to compliment the user about something before suggesting anything. 😉